### PR TITLE
Feature/eap local pf account

### PR DESCRIPTION
**Pull request by fdurand**
For review

### CODE REVIEW

File: conf/documentation.conf [hunk 1]
```diff
@@ -445,6 +445,22 @@ Sponsors requesting access and access confirmation emails are CC'ed to those
 addresses. Multiple destinations can be comma separated.
 EOT
 
+[guests_self_registration.create_local_account_on_email_reg]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Create a local account after guest email registration. This will allow the user to authenticate on
+the status page or be able to authenticate on a eap connection see (packetfence-tunnel)
```
Comment (extrafu) [inline conf/documentation.conf]: over an EAP connection

File: conf/documentation.conf [hunk 2]
```diff
@@ -445,6 +445,22 @@ Sponsors requesting access and access confirmation emails are CC'ed to those
...
+EOT
+
+[guests_self_registration.create_local_account_on_sponsor_reg]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Create a local account after guest sponsor registration. This will allow the user to authenticate on
+the status page or be able to authenticate on a eap connection see (packetfence-tunnel)
```
Comment (extrafu) [inline conf/documentation.conf]: on a eap connection => over an EAP connection

File: conf/pf.conf.defaults [hunk 3]
```diff
@@ -200,7 +200,20 @@ preregistration=disabled
 # Sponsors requesting access and access confirmation emails are CC'ed to this
 # address. Multiple destinations can be comma separated.
 sponsorship_cc=
-
+#
+# guests_self_registration.create_local_account_on_email_reg
+#
+# Create a local user account on email registration to allow the user
+# to manage his device on the status page or to use the credential
+# on a eap connection
```
Comment (extrafu) [inline conf/pf.conf.defaults]: same

File: conf/pf.conf.defaults [hunk 4]
```diff
@@ -200,7 +200,20 @@ preregistration=disabled
...
+create_local_account_on_email_reg=disabled
+#
+# guests_self_registration.create_local_account_on_sponsor_reg
+#
+# Create a local user account on sponsor registration to allow the user
+# to manage his device on the status page or to use the credential
+# on a eap connection
```
Comment (extrafu) [inline conf/pf.conf.defaults]: same

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 5]
```diff
@@ -3066,6 +3066,18 @@ msgid ""
   "addresses. Multiple destinations can be comma separated."
 msgstr ""
 
+# conf/documentation.conf (guests_self_registration.create_local_account_on_email_reg)
+msgid ""
+  "Create a local account after guest email registration. This will allow the user to authenticate on"
+  "the status page or be able to authenticate on a eap connection see (packetfence-tunnel)"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: same thing

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 6]
```diff
@@ -3066,6 +3066,18 @@ msgid ""
...
+msgstr ""
+
+# conf/documentation.conf (guests_self_registration.create_local_account_on_sponsor_reg)
+msgid ""
+  "Create a local account after guest sponsor registration. This will allow the user to authenticate on"
+  "the status page or be able to authenticate on a eap connection see (packetfence-tunnel)"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: same thing

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 7]
```diff
@@ -4274,6 +4286,14 @@ msgstr "Preregistration"
 msgid "guests_self_registration.sponsorship_cc"
 msgstr "Sponsorship CC"
 
+# conf/documentation.conf
+msgid "guests_self_registration.create_local_account_on_email_reg"
+msgstr "Create user after email reg"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: reg -> registration

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 8]
```diff
@@ -4274,6 +4286,14 @@ msgstr "Preregistration"
...
+
+# conf/documentation.conf
+msgid "guests_self_registration.create_local_account_on_sponsor_reg"
+msgstr "Create user after sponsor reg"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: reg -> registration

Comment (fdurand) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: Too long in the admin GUI

File: html/pfappserver/lib/pfappserver/I18N/en.po (hunk 7 above)
Comment (fdurand) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: Too long in the admin GUI

File: conf/documentation.conf [hunk 9]
```diff
@@ -445,6 +445,22 @@ Sponsors requesting access and access confirmation emails are CC'ed to those
 addresses. Multiple destinations can be comma separated.
 EOT
 
+[guests_self_registration.create_local_account_on_email_reg]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Create a local account after guest email registration. This will allow the user to authenticate on
```
Comment (extrafu) [inline conf/documentation.conf:452]: after guest email/SMS registration ?

File: conf/pf.conf.defaults [hunk 10]
```diff
@@ -200,7 +200,20 @@ preregistration=disabled
 # Sponsors requesting access and access confirmation emails are CC'ed to this
 # address. Multiple destinations can be comma separated.
 sponsorship_cc=
-
+#
+# guests_self_registration.create_local_account_on_email_reg
+#
+# Create a local user account on email registration to allow the user
```
Comment (extrafu) [inline conf/pf.conf.defaults:206]: on email/SMS registration?

File: conf/documentation.conf (hunk 9 above)
Comment (fdurand) [inline conf/documentation.conf:452]: Just email, for SMS we are using the phone number and the pin code (always created).

File: conf/pf.conf.defaults (hunk 10 above)
Comment (fdurand) [inline conf/pf.conf.defaults:206]: Just email, for SMS we are using the phone number and the pin code (always created).

**Pull request merged**